src/gui/layout/qgslayoutunitscombobox.h

```diff
@@ -82,10 +82,18 @@ class GUI_EXPORT QgsLayoutUnitsComboBox : public QComboBox
 
   signals:
 
+#ifndef SIP_RUN
+
+    /**
+     * Emitted when the \a unit is changed.
+     */
+    void unitChanged( Qgis::LayoutUnit unit );
+#endif
+
     /**
      * Emitted when the \a unit is changed.
      */
-    void changed( Qgis::LayoutUnit unit );
+    void changed( int );
```

Comment:
add a note to drop in QGIS 4?

Reply:
Well, it depends on whether sip fixes this. (It's unrelated to the qgis compatibility code)